cf/crypto.m4

```diff
@@ -135,6 +135,17 @@ if test "$crypto_lib" = "unknown"; then
 
 fi
 
+AC_ARG_WITH(pkcs11-module,
```

Comment:
Ahh!  Got it.  Thanks.  Also, there's no place in sight where we can store any configuration.